ref(node/serverless): Add note about performance impact of auto discovery of integrations (#7245)
getsentry/sentry-docs: src/platforms/node/common/performance/instrumentation/automatic-instrumentation.mdx

<Note>

If you're adopting Performance in a high-throughput environment, we recommend testing prior to deployment to ensure that your service's performance characteristics maintain expectations.

> Note that calling `Sentry.autoDiscoverNodePerformanceMonitoringIntegrations` might degrade startup performance of your app. This is especially a concern in Serverless functions. If you're experiencing this, manually add [database integrations](../../database/) of your choice instead (see example below).

Reviewer: ```suggestion
Note, that calling `Sentry.autoDiscoverNodePerformanceMonitoringIntegrations` might degrade startup performance of your app. This is especially a concern for Serverless functions. If you're experiencing this, add [database integrations](../../database/) of your choice manually (see example below).
```

> Note that calling `Sentry.autoDiscoverNodePerformanceMonitoringIntegrations` might degrade startup performance of your app. This is especially a concern in Serverless functions. If you're experiencing this, manually add [database integrations](../../database/) of your choice instead (see example below).

Reviewer 2: Also, isn't it pretty much 100% guaranteed that this function is slow?

Author: Yes I think so. Let me reword this slightly.

    // enable HTTP calls tracing
    new Sentry.Integrations.Http({ tracing: true }),
    // Automatically instrument Node.js libraries and frameworks

>     // (This might degrade startup performance)

Reviewer 2: Idk about this line. It just says it may (100% will) degrade performance and there's no clear indicator of what the alternative is.

Maybe a slight reword?:

```suggestion
    // (This function is slow because of file I/O, consider manually adding additional integrations instead)
```
